plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -296,6 +317,38 @@ class TaskTable extends React.Component {
     );
   }
 
+  renderRegion(prop, task) {
+    if (!task.region) {
+      return "Primary";
```

Comment:
@bstavroulakis Let's use the same fallback as in [`NodesTable.js#L119`](https://github.com/dcos/dcos-ui/blob/master/plugins/nodes/src/js/components/NodesTable.js#L119) to be consistent. /cc @Poltergeist

Reply:
@orlandohohmeier @Poltergeist I'll have a definite answer from product to see what to do here.